tools/qualimap.wdl

```diff
@@ -52,45 +56,53 @@ task rnaseq {
     input {
         File bam
         File gencode_gtf
-        String outdir = "qualimap_rnaseq"
+        String prefix = basename(bam, ".bam") + "_qualimap_rnaseq_results"
         Int memory_gb = 16
         Int? disk_size_gb
         Int max_retries = 1
-        String strand = ""
-        String inferred = ""
+        String provided_strand = ""
+        String inferred_strand = ""
     }
 
-    String stranded = if (strand != "") then 
-                        if (strand == "reverse") then "strand-specific-reverse" else
-                        if (strand == "yes") then "strand-specific-forward" else
-                        if (strand == "no") then "non-strand-specific"
+    String out_directory = select_first([prefix, "qualimap_rnaseq_results"])
+    String out_tar_gz_file = out_directory + ".tar.gz"
+    String stranded = if (provided_strand != "") then 
+                        if (provided_strand == "stranded-reverse") then "strand-specific-reverse" else
+                        if (provided_strand == "stranded-forward") then "strand-specific-forward" else
+                        if (provided_strand == "unstranded") then "non-strand-specific"
                         else "unknown-strand"
                       else 
-                        if (inferred == "Stranded-Reverse") then "strand-specific-reverse" else
-                        if (inferred == "Stranded-Forward") then "strand-specific-forward" else 
-                        if (inferred == "Unstranded") then "non-strand-specific" else
+                        if (inferred_strand == "Stranded-Reverse") then "strand-specific-reverse" else
+                        if (inferred_strand == "Stranded-Forward") then "strand-specific-forward" else 
+                        if (inferred_strand == "Unstranded") then "non-strand-specific" else
                         "unknown-strand" # this will intentionally cause htseq to error. You will need to manually specify
                                          # in this case
 
     Int java_heap_size = ceil(memory_gb * 0.9)
-
     Float bam_size = size(bam, "GiB")
     Float gencode_gtf_size = size(gencode_gtf, "GiB")
     Int disk_size = select_first([disk_size_gb, ceil(((bam_size + gencode_gtf_size) * 12) + 10)])
  
     command {
-        qualimap rnaseq -bam ${bam} -gtf ${gencode_gtf} -outdir ${outdir} -oc qualimap_counts.txt -p ${stranded} -pe --java-mem-size=${java_heap_size}G
+        qualimap rnaseq -bam ~{bam} \
+                        -gtf ~{gencode_gtf} \
+                        -outdir ~{out_directory} \
+                        -oc qualimap_counts.txt \
+                        -p ~{stranded} \
+                        -pe \
```

Comment:
I think we need to make this a global workflow option, some RNA-seq data is single end. But we can default ours all to `true`.